storage/statedb/iterator.go

```diff
@@ -149,7 +150,7 @@ func (e seekError) Error() string {
 }
 
 func newNodeIterator(trie *Trie, start []byte) NodeIterator {
-	if trie.Hash() == emptyState {
+	if trie.Hash() == types.EmptyCodeHash {
```

Comment:
Can you create `emptyTrieEntry` here in `iterator.go` just for here?

- Geth has EmptyRootHash at this location (https://github.com/ethereum/go-ethereum/blob/c7a8bcecbed7b371f0d6ec97ab1a1ae33aa3ca85/trie/iterator.go#L165). But I don't want to change the existing logic
- `trie.Hash()` is definitely not a code hash
- Therefore we need another name for `0xc5d2..`. Previously it said "emptyState is the known hash of an empty state trie entry.", so emptyTrieEntry is a descriptive name.